lib/Sema/SemaDecl.cpp

```diff
@@ -13083,9 +13083,13 @@ ExprResult Sema::VerifyBitField(SourceLocation FieldLoc,
     // Handle incomplete types with specific error.
     if (RequireCompleteType(FieldLoc, FieldTy, diag::err_field_incomplete))
       return ExprError();
-    if (FieldName)
-      return Diag(FieldLoc, diag::err_not_integral_type_bitfield)
-        << FieldName << FieldTy << BitWidth->getSourceRange();
+    if (FieldName) {
+      if (getLangOpts().CheckedC && FieldTy->isPointerType())
+        return Diag(BitWidth->getExprLoc(), diag::err_expected_bounds_expr_for_member);
```

Comment:
>BitWidth->getExprLoc(), diag::err_expected_bounds_expr_for_member [](start = 20, length = 65)

Following the logic below, should we also append 'FieldName', 'FieldTy', and the source range for whatever was after the colon?